python/lsst/ctrl/execute/slurmPlugin.py

```diff
@@ -137,6 +137,11 @@ def loadSlurm(self, name, platformPkgDir):
             self.allocationFileName
         )
 
+        if self.opts.packnodes is None:
+            self.defaults["PACK_BLOCK"] = "#"
+        else:
+            self.defaults["PACK_BLOCK"] = "Rank = TotalCpus - Cpus"
```

Comment:
Maybe for non-HTCondor experts put a brief comment here explaining why this difference will pack.